src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -135,9 +135,12 @@ void FixContext(PCONTEXT pContextRecord)
         pContextRecord->reg = (value);                                                      \
     } while (0)
 
-#ifdef TARGET_X86
+#if defined(TARGET_X86)
     size_t resumeSp = EECodeManager::GetResumeSp(pContextRecord);
     FIXUPREG(Esp, resumeSp);
+#elif defined(TARGET_LOONGARCH64)
+    int cpuFeatures = minipal_getcpufeatures();
+    pContextRecord->ContextFlags |= cpuFeatures;
```

Comment:
The code should not be assuming that the flags returned by `minipal_getcpufeatures` match the CONTEXT flags.